core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -594,13 +618,15 @@ private void copyLogSegment(UnifiedLog log, LogSegment segment, long nextSegment
             LogSegmentData segmentData = new LogSegmentData(logFile.toPath(), toPathIfExists(segment.lazyOffsetIndex().get().file()),
                     toPathIfExists(segment.lazyTimeIndex().get().file()), Optional.ofNullable(toPathIfExists(segment.txnIndex().file())),
                     producerStateSnapshotFile.toPath(), leaderEpochsIndex);
+            brokerTopicStats.topicStats(log.topicPartition().topic()).remoteWriteRequestRate().mark();
```

Comment:
Can you add this to the description of the pull request as well? I suspect this is a previously missed metric, but I wasn't expecting it when I was reviewing the pull request.

Reply:
Oh, this is based on the change in https://github.com/apache/kafka/pull/13944/files, that is why there are changes not listed in the description, got it.

Reply:
Sorry, didn't notice you're reviewing it already. Yes. you're right. Now the PR is updated to only tests left.